rodan-main/code/rodan/jobs/MEI_encoding/build_mei_file.py

```diff
@@ -586,15 +635,48 @@ def build_mei(pairs: List[Tuple[List[dict], dict]], classifier: dict, width_cont
 
                 continue
 
+
             cur_staff = int(glyph['staff'])
 
+            # if multi column and next system is new column, add new cb
+            if is_multi_column:
+                if staff_to_column[cur_staff] > prev_column:
+                    # add cb to layer
+                    zoneId = generate_zone(surface, previous_cb["bb"])
+                    previous_cb["cb"].set("facs", "#" + zoneId)
+
+                    #start new cb
+                    prev_column = staff_to_column[cur_staff]
+                    cb = new_el("cb")
+                    cb.set("n", str(staff_to_column[cur_staff] + 1))
+                    previous_cb["cb"] = cb
+                    previous_cb["bb"] = None
+                    layer.append(cb)
+

```

Comment:
all this stuff could be refactored into a separate method (a `start_new_cb` or something) so that it isn't right in the middle of everything

Reply:
this block is setting variables that are used else where in the loop, I'm not sure it makes sense to move it to its own function